sqlmesh/core/config/root.py

```diff
@@ -50,6 +50,7 @@ class Config(BaseConfig):
         username: Name of a single user who should receive approvals/notification, instead of all users in the `users` list.
         pinned_environments: A list of development environment names that should not be deleted by the janitor task.
         loader: Loader class used for loading project files.
+        loader_kwargs: Key-value arguments to pass to the loader instance.
```

Comment:
is this dbt only?

Reply:
should we respect it in sqlmesh?

Reply:
yeah, why not? It works for both

Reply:
we just don't have anything for sqlmesh right now